### PR TITLE
Adding environment.yml for binder

### PR DESCRIPTION
**Pull request by mortonjt**
This does the following

1. Adding environment.yml for binder
2. Fixing warnings
3. Deprecating diamondtree in favor of radialplot

### CODE REVIEW

**Comment by mortonjt** [comment]
@antgonza would you like to merge this?  This is required to get binder up and running.

**Comment by mortonjt** [comment]
We'll also push out another minor release after this - still testing out the new set of dependencies.

File: gneiss/balances.py [hunk 1]
```diff
@@ -34,9 +34,8 @@
 try:
     from ete3 import Tree, TreeStyle
     from gneiss.layouts import default_layout
-except ImportError:
-    raise ImportWarning('ete3 is not installed.  '
-                        'ETE3 style visualizations will not be available.')
+except:
```
Comment (antgonza) [inline gneiss/balances.py]: why this change? perhaps printing the error will be useful for users

File: gneiss/layouts.py [hunk 2]
```diff
@@ -7,9 +7,8 @@
 # ----------------------------------------------------------------------------
 try:
     from ete3 import faces, AttrFace, CircleFace, BarChartFace
-except ImportError:
-    raise ImportWarning('ete3 is not installed.  '
-                        'ETE3 style visualizations will not be available.')
+except:
```
Comment (antgonza) [inline gneiss/layouts.py]: same: why and printing error useful for user?

File: gneiss/plot/_blobtree.py [hunk 3]
```diff
@@ -5,225 +5,9 @@
...
-            return x[node.name], True
-        elif x is None:
-            return "", False
-        else:
-            raise TypeError("color type %s not supported." % type(x).__name__)
-    except KeyError:
-        # Use default if the color isn't specified
-        c = default_color
-        return c, False
+    warnings.warn("`diamondtree` has been deprecated.  "
```
Comment (antgonza) [inline gneiss/plot/_blobtree.py]: if this is beta, is it really worth to keep this or simply remove?

File: ipynb/cfstudy/cfstudy.ipynb [hunk 4]
```diff
@@ -11,11 +11,79 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 4,
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
```
Comment (antgonza) [inline ipynb/cfstudy/cfstudy.ipynb]: do you really want to add all these tracebacks to your notebook?

File: gneiss/balances.py (hunk 1 above)
Comment (mortonjt) [inline gneiss/balances.py]: This will still print an error if the function is called. 

Apparently `raise ImportWarning` raises an exception - which breaks the entire CLI interface.
Another options is to use `warnings.warn(...)`.  However, this isn't what we want, since this warning will be printed every time a command line call is made.

Comment (mortonjt) [inline gneiss/balances.py]: Here's an example of what this error looks like

screen shot 2017-03-07 at 7 06 10 am

Do you think this is acceptable?

File: gneiss/plot/_blobtree.py (hunk 3 above)
Comment (mortonjt) [inline gneiss/plot/_blobtree.py]: Let's remove it.

File: gneiss/balances.py (hunk 1 above)
Comment (antgonza) [inline gneiss/balances.py]: yup, thanks for clarifying.

**Comment by mortonjt** [comment]
@antgonza thank you for reviewing  this one - that could have been bad if we merged in all of the tracebacks.  And I think the import errors are a little more clear now.

I think it should ok once the tests pass.  Thanks!
